messaging/src/main/java/org/axonframework/eventhandling/pooled/WorkPackage.java

```diff
@@ -98,6 +98,7 @@ class WorkPackage {
     private final AtomicBoolean scheduled = new AtomicBoolean();
     private final AtomicReference<CompletableFuture<Exception>> abortFlag = new AtomicReference<>();
     private final AtomicReference<Exception> abortException = new AtomicReference<>();
+    private final Runnable clearCacheFunction;
```

Comment:
If we rename the operation on the `EventHandlerInvoker`, I anticipate we rename the field name too.
Perhaps something like `shutdownAction` (as present in the `Coordinator`) or `abortAction` would make sense.

Reply:
It's now done from the `Coordinator` on releasing the token. So no changes to the `Workpackage` are needed anymore.